### PR TITLE
hindi translation

### PR DESCRIPTION
**Pull request by AceToXiC**
hello i am toxci from INDIA i reallly liked you concept. so I think that many indian will like this concept therefor i hav translated the website in hindi

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="hindi">
```
Comment (tulir) [inline index.html:2]: The language code for Hindi is `hi`

File: index.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 <head>
-  <title>Don't ask to ask, just ask</title>
-  <meta property="og:title" content="Don't ask to ask, just ask" />
+  <title>पूछने के लिए मत पूछो, बस पूछो</title>
+  <meta property="og:title" content="पूछने के लिए मत पूछो, बस पूछो" />
   <meta property="og:type" content="website" />
   <meta property="og:url" content="https://dontasktoask.com" />
   <meta property="og:locale" content="en_US" />
```
Comment (tulir) [inline index.html:8]: This should also be changed, I think the code is `hi_IN`

File: index.html [hunk 3]
```diff
@@ -14,75 +14,75 @@
...
       <span class="name">Foobar123:</span>
       <p class="message">
-        Any Java experts around?
+        क्या कोई जावा विशेषज्ञ है?
       </p>
     </blockquote>
     <p>
-      This is bad form, for several reasons. What the person is
+      यह कई कारणों से खराब रूप है। व्यक्ति वास्तव में यहाँ क्या पूछ रहा है,
       <em>actually</em> asking here is,
```
Comment (tulir) [inline index.html:28]: This bit isn't translated

**Review by tulir — CHANGES_REQUESTED** [review]
Hi, thanks for the translation!

Translations should be placed in a new directory with the language code instead of modifying the root `index.html` directly. In this case, you should put it in `hi/index.html` (`hi` is the ISO 639-1 language code for Hindi). The readme doesn't need to be translated at all.

File: index.html (hunk 1 above)
Comment (AceToXiC) [reply index.html:2]: ok i'll do it

**Comment by AceToXiC** [comment]
Hello,
I have made the changes you requested
Hope you like it.

Thank you.

**Pull request closed without merging**